server/database/migrations/20190729172658-create-user.js

```diff
@@ -0,0 +1,64 @@
+module.exports = {
```

Comment:
Why do we have user creation here

Reply:
I created it to test the login route with a user seeded into the database